Solutions/Votiro/Analytic Rules/VotiroFileBlockedInEmail.yaml

```diff
@@ -0,0 +1,40 @@
+id: 0b8b91de-c63e-4bc2-b5f4-b15d3b379ec9
+name: Votiro - File Blocked in Email
+description: |
+ 'The analytic rule is designed to identify when an email is blocked by Votiro Sanitization Engine policy. The rule generates an alert when an email is blocked after Sanitization process which is not password protected and has a from field meaning its a valid email. More information in terms of details about the blocked attachment, such as the file name and hash, and information about the email, such as the sender and recipient(s). The alert also includes information about the Votiro policy that blocked the attachment and provides a link to additional details about the incident.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Votiro
+    dataTypes:
+     - CommonSecurityLog 
+queryFrequency: 10m
+queryPeriod: 10m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
+  - DefenseEvasion
+  - Impact
+  - InitialAccess
+relevantTechniques:
+  - T0885
+  - T1036
+  - T1027
+  - T1486
+  - T1566
+query: let Votiro_view  = view () {
+  VotiroEvents
```

Comment:
Same logic is used in both the queries. What is the difference here.
Also, there is no entity mapping present here which is ideal to be used in Analytical Rule.

Reply:
Hi @devikamehra ,
The functionality of both is similar, except for the fields after "extend" as the data retrieval looks different.

As for the "entity mapping" part we are looking into it.
Thanks,
Sambhrant